src/sidebar/components/hooks/use-element-should-close.js

```diff
@@ -24,6 +24,21 @@ export default function useElementShouldClose(
   isOpen,
   handleClose
 ) {
+  /**
+   *  Helper to return the underlying node object whether
+   *  `closeableEl` is attached to an HTMLNode or Preact component.
+   *
+   *  @param {Preact ref} closeableEl
+   *  @returns {HTMLNode}
+   */
+
+  const getCurrentNode = closeableEl => {
+    // if base is present, assume its a preact component
```

Comment:
The `base` property is undocumented, although I have been told by the devs that it is considered a public API ([Slack comment](https://preact.slack.com/archives/C3NMBSJKH/p1578589918070400)). Since this is a Preact-specific API (as opposed to being part of the core APIs that Preact/React share), I suggest adding a short note about what exactly `base` refers to (the first rendered DOM child node of the component).